src/elements/content-picker/README.md

```diff
@@ -40,6 +40,7 @@ var ContentPicker = require('./ContentPicker').default;
 | onCancel | function |  | Callback function for when the cancel button is pressed. |
 | onChoose | function |  | Callback function for when the choose button is pressed. |
 | requestInterceptor | function | | *See the [developer docs](https://developer.box.com/guides/embed/ui-elements/picker/#Options).* |
+| renderCustomActionButtons | function | React.Node | Render prop that renders custom action buttons in the footer. |
```

Comment:
you can remove `React.Node` since that column is "Default" - you can leave it empty since there is no default value for renderCustomActionButtons